M2/.gitignore

```diff
@@ -44,9 +44,12 @@
 /libraries/missing/src/Makefile
 /libraries/Makefile.library
 /Macaulay2/bin/M2
-/Macaulay2/c/.gdbinit
-/Macaulay2/m2/.gdbinit
+/Macaulay2/c/.gdbinit*
+/Macaulay2/m2/.gdbinit*
```

Comment:
We don't want to ignore M2/Macaulay2/m2/.gdbinit.dan and M2/Macaulay2/m2/.gdbinit.mike

Reply:
I see. I reverted that.